packages/styles/src/components/sizing.scss

```diff
@@ -55,6 +55,21 @@
   }
 }
 
+// Post breakpoints width - e.g. w-sm-100
+@each $breakpoint in map-keys(breakpoints.$grid-breakpoints) {
+  @include media-breakpoint-up($breakpoint) {
+    $infix: if($breakpoint == 'xs', '', '-#{$breakpoint}');
```

Comment:
Since you are creating them again here, it would be good to use the [utility API](https://getbootstrap.com/docs/5.3/utilities/api/#remove-utilities) and remove the width classes generated by Bootstrap.